charts/patroni-spilo/values.yaml

```diff
@@ -0,0 +1,133 @@
+replicaCount: 5
+
+image:
+  # Image was built from
+  # https://github.com/zalando/spilo/tree/master/postgres-appliance
+  repository: registry.opensource.zalan.do/acid/spilo-10
+  tag: 1.5-p5
+  pullPolicy: IfNotPresent
+  pullSecrets: []
+  #- name: abc
+
+# Credentials used by Patroni
+# https://github.com/zalando/patroni/blob/master/docs/SETTINGS.rst#postgresql
+# https://github.com/zalando/spilo/blob/master/ENVIRONMENT.rst
+credentials:
+  random: false
```

Comment:
I see a lot of random passwords in the files changed. Does this override all of that then?